backend/log/log.go

```diff
@@ -0,0 +1,54 @@
+package log
+
+import (
+	hclog "github.com/hashicorp/go-hclog"
+)
+
+var defaultLogger Logger
+
+func init() {
+	defaultLogger = New()
+}
+
+// Logger the main Logger interface.
+type Logger interface {
+	Debug(msg string, args ...interface{})
+	Info(msg string, args ...interface{})
+	Warn(msg string, args ...interface{})
+	Error(msg string, args ...interface{})
+}
+
+// New creates a new logger.
+func New() Logger {
+	return &hclogWrapper{
+		logger: hclog.New(&hclog.LoggerOptions{
+			Level:      hclog.LevelFromString("DEBUG"),
```

Comment:
add comment about DEBUG and Grafana-server filtering.

Reply:
Done